registry/handlers/app.go

```diff
@@ -596,24 +596,19 @@ func (app *App) configureSecret(configuration *configuration.Configuration) {
 func (app *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close() // ensure that request body is always closed.
 
-	// Instantiate an http context here so we can track the error codes
-	// returned by the request router.
-	ctx := defaultContextManager.context(app, w, r)
```

Comment:
App context provenance is maintained at https://github.com/docker/distribution/blob/master/registry/handlers/app.go#L637. What exactly did this break?